### DISCUSSION TITLE
Is HVAC unable to verify a re-key operation with the recovery_key=True flag?

### DISCUSSION THREAD

**Discussion by Ar-personal** [hvac/hvac#1094] [Q&A]
The HVAC code usually swaps out the api path for re-key operations from api_path = "/v1/sys/rekey/<opertation>" to api_path = "/v1/sys/rekey-recovery-key/<operation> However the rekey_verify function only has the former option, am I missing something here or is this incomplete?

def rekey_verify(self, key, nonce):
    ...
    api_path = "/v1/sys/rekey/verify"

**Comment by briantist** [comment]
Hi @Ar-personal , it seems like this functionality is not available in `hvac` right now, so I've opened #1097 based on this discussion and will close this out to track it in the issue. Additionally, a PR was just opened that seems like it will address this: #1095
